pkg/operator/controllers/guardrails/policies/README.md

```diff
@@ -0,0 +1,305 @@
+      including, ["^kube.*|^openshift.*|^default$|^redhat.*|^com$|^io$|^in$"]
+spec:
+  crd:
+    spec:
+      names:
+        kind: AROPrivilegedNamespace
+      validation:
+        # Schema for the `parameters` field
+        openAPIV3Schema:
+          type: object
+          description: >-
+            Disallows creating, updating or deleting resources in privileged namespaces.
+            including, ["^kube.*|^openshift.*|^default$|^redhat.*|^com$|^io$|^in$"]
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+{{ file.Read "gktemplates-src/aro-deny-privileged-namespace/src.rego" | strings.Indent 8 | strings.TrimSuffix "\n" }}
+```
+
+
+* Create the src.rego file in the same folder, howto https://www.openpolicyagent.org/docs/latest/policy-language/, example:
+```
+package aroprivilegednamespace
+
+violation[{"msg": msg, "details": {}}] {
+  input_priv_namespace(input.review.object.metadata.namespace)
+  msg := sprintf("Operation in privileged namespace %v is not allowed", [input.review.object.metadata.namespace])
+}
+
+input_priv_namespace(ns) {
+  any([ns == "default",
+  ns == "com",
+  ns == "io",
+  ns == "in",
+  startswith(ns, "openshift"),
+  startswith(ns, "kube"),
+  startswith(ns, "redhat")])
+}
+```
+* Create src_test.rego for unit tests in the same foler, which will be called by test.sh, howto https://www.openpolicyagent.org/docs/latest/policy-testing/, example:
+```
+package aroprivilegednamespace
+
+test_input_allowed_ns {
+  input := { "review": input_ns(input_allowed_ns) }
+  results := violation with input as input
+  count(results) == 0
+}
+
+test_input_disallowed_ns1 {
+  input := { "review": input_review(input_disallowed_ns1) }
+  results := violation with input as input
+  count(results) == 1
+}
+
+input_ns(ns) = output {
+  output = {
+    "object": {
+      "metadata": {
+        "namespace": ns
+      }
+    }
+  }
+}
+
+input_allowed_ns = "mytest"
+
+input_disallowed_ns1 = "openshift-config"
+```
+
+* Create [Constraint](https://open-policy-agent.github.io/gatekeeper/website/docs/howto/#constraints) for the policy, example:
+
+```yaml
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: AROPrivilegedNamespace
+metadata:
+  name: aro-privileged-namespace-deny
+spec:
+  enforcementAction: {{.Enforcement}}
+  match:
+    kinds:
+      - apiGroups: [""]
+        kinds: [
+        "Pod",
+        "Secret",
+        "Service",
+        "ServiceAccount",
+        "ReplicationController",
+        "ResourceQuota",
+        ]
+      - apiGroups: ["apps"]
+        kinds: ["Deployment", "ReplicaSet", "StatefulSet", "DaemonSet"]
+      - apiGroups: ["batch"]
+        kinds: ["Job", "CronJob"]
+      - apiGroups: ["rbac.authorization.k8s.io"]
+        kinds: ["Role", "RoleBinding"]
+      - apiGroups: ["policy"]
+        kinds: ["PodDisruptionBudget"]
+```
+
+## Test the rego
+
+* install opa cli, refer https://github.com/open-policy-agent/opa/releases/
```

Comment:
This (and other lines here) look like we need a change to our onboarding docs to set up dev environments with all required tooling.